views/employees.py

```diff
@@ -102,10 +102,14 @@ def employee_uri():
             if empl_id:
                 for x in empl_id:
                     employee = query.remove_employee_by_id(x)
+                #     role_deleted = query.remove_employee_role_by_id(x)
+                #     print(role_deleted)
+                # if employee and role_deleted: 
                 return ("Success", 200)
             if employee:
```

Comment:
Deleting multiple employees should be done in the `/api/employees` endpoint, not this one. This one is just for operations on individual employees.